ckan/ckanext/ckanext-ytp_main/ckanext/ytp/templates/general_search/snippets/post_pagination.html

```diff
@@ -0,0 +1,105 @@
+{% asset 'ytp_resources/post_pagination_js' %}
+
+
+{%- set current_page = 1 if not page else page -%}
+{%- set plus_minus = 2 -%}
```

Comment:
This variable isn't quite self descriptive named, so probably rename it or add some comment to describe what is it used for

Reply:
Resolved in [92a479a](https://github.com/vrk-kpa/opendata/pull/1774/commits/92a479a6671ee9d1f0071c475d1135bf7a17fbd2)